src/Sylius/Bundle/AdminBundle/templates/product/form/sections/media/images.html.twig

```diff
@@ -37,9 +37,11 @@
                 <div class="mb-3">
                     {{ form_row(image_form.file, sylius_test_form_attribute('file')) }}
                 </div>
-                <div class="mb-3">
-                    {{ form_row(image_form.position, sylius_test_form_attribute('position')) }}
-                </div>
+                {% if image_form.position is defined %}
```

Comment:
this is probably a similar case as with https://github.com/Sylius/Sylius/blob/2.0/src/Sylius/Bundle/AdminBundle/templates/product_attribute/form/general/position.html.twig